docs/reference/commandline/daemon.md

```diff
@@ -64,6 +64,7 @@ weight = -1
       --tlscert="~/.docker/cert.pem"         Path to TLS certificate file
       --tlskey="~/.docker/key.pem"           Path to TLS key file
       --tlsverify                            Use TLS and verify the remote
+      --no-default-tlsverify                 Turns off default remote TLS verification
```

Comment:
Thanks! Sure! It seems I was to much focused on the advice given in the description of this enhancement request.